partition_snapshot_row_cursor.hh

```diff
@@ -428,7 +587,9 @@ public:
     // Assumes the snapshot is evictable and not populated by means other than ensure_entry_if_complete().
     // Subsequent calls to ensure_entry_if_complete() must be given strictly monotonically increasing
     // positions unless iterators are invalidated across the calls.
+    // The cursor must not be a reversed-order cursor.
     std::optional<ensure_result> ensure_entry_if_complete(position_in_partition_view pos) {
+        assert(!_reversed); // latest_i is unreliable
```

Comment:
Wouldn't it be better to use `on_internal_error`?

Reply:
That would crash the query but not the node.

Reply:
Yes